src/ucx_van.h

```diff
@@ -519,8 +589,9 @@ class UCXVan : public Van {
     msg->meta.sender = buf.sender;
     msg->meta.recver = my_node_.id;
 
+    // for pull request, we still need to set the key
```

Comment:
can you clarify what this comment means?

Reply:
It's a comment for the `msg->meta.push && !msg->meta.request` condition (we don't return directly for pull request).